packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4933,10 +4933,7 @@ export default class Meeting extends StatelessWebexPlugin {
           payload: {canProceed: false},
           options: {
             meetingId: this.id,
-            error: {
-              // TODO: adapt this code to work in separate JIRA
-              error: 'this is not really mapped, this error is very contextual.',
-            },
+            parsedError: NewMetrics.callDiagnosticMetrics.getErrorPayloadForClientErrorCode(2004),
```

Comment:
how do you know `2004` is the error code? what is the meaning of `2004`

Reply:
I guess i can create a constant with this. This was the client error code that we sent in the old implementation. See here: https://github.com/webex/webex-js-sdk/blob/beta/packages/%40webex/plugin-meetings/src/meeting/index.ts#L4722

Reply:
please create a constant

Reply:
added